ikuseiGmbH/smart-village-app-app
# fix: POIs open times issue

- fixed an issue with POIs showing as open if the day is not selected on days when POIs are closed

SVA-925

## How to test:
the cause of the problem:
some POIs do not have the day and time selected. if the day and time are not selected, the code automatically thinks that the POI is open because it does not check the day.

- go to POI with id `38` in CMS
- make the `geöffnet` of the current day false
- please make sure you select the correct day via the CMS when reviewing
- in the app go to `Touren und Orte` -> `Gastronomie` -> `"Panda Bistro"`
- check whether the selected POI is open or closed

## Screenshots:

![image](https://user-images.githubusercontent.com/11755668/223976139-20f4c696-0763-415a-bd8c-a6a10747a90f.png)

|before|after|
|--|--|
![Simulator Screen Shot - iPhone 14 Pro Max - 2023-03-09 at 10 07 47](https://user-images.githubusercontent.com/11755668/223973861-f0dbc97e-c02e-4717-861f-1425a560db4d.png) | ![image](https://user-images.githubusercontent.com/11755668/223975934-a7212904-6010-4f1d-95e5-435b9b51a13f.png)
